hbase-server/src/test/java/org/apache/hadoop/hbase/StartMiniClusterOption.java

```diff
@@ -76,6 +76,11 @@
    */
   private Class<? extends MiniHBaseCluster.MiniHBaseClusterRegionServer> rsClass;
 
+  /**
+   * Number of compaction servers to start up.
+   */
+  private final int numCompactionServers;
```

Comment:
Why final?

Reply:
Variables in `StartMiniClusterOption` are all declared final, the `StartMiniClusterOption` is useless after minicluster startup.